changelogs/fragments/filetree_create_inventories_source_bugfix.yml

```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - port https://github.com/redhat-cop/infra.controller_configuration/pull/39 to this repo
```

Comment:
We need to be more descriptive than this. Can we just expand this to what the actual fix is please. This is now the de-facto standard repo for this code so we shouldn't be referring to the repo which is just there for backports